hadoop/src/main/java/org/apache/carbondata/hadoop/api/CarbonTableInputFormat.java

```diff
@@ -143,11 +143,15 @@ protected CarbonTable getOrCreateCarbonTable(Configuration configuration) throws
     SegmentStatusManager segmentStatusManager = new SegmentStatusManager(identifier);
     SegmentStatusManager.ValidAndInvalidSegmentsInfo segments =
         segmentStatusManager.getValidAndInvalidSegments(loadMetadataDetails);
-
-    if (getValidateSegmentsToAccess(job.getConfiguration())) {
+    // if for streaming table only access is streaming table is enabled access then access
+    // only streaming segments this will be
```

Comment:
This comment is not clear. Kindly re write it for better understanding

Reply:
ok